.ci_support/win_64_.yaml

```diff
@@ -0,0 +1,18 @@
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+gmp:
+- '6'
+m2w64_c_compiler:
+- gcc
+m2w64_c_compiler_version:
+- '13'
+m2w64_c_stdlib:
+- m2w64-sysroot
+m2w64_c_stdlib_version:
+- '12'
```

Comment:
Do we want _any_ of the `.ci_support` tree?  Particularly as it is referencing `m2w64` which we are deprecating/wanting to switch away from.

Reply:
it doesn't do anything, it's just conda-forge CI stuff.
Including it makes it easier to sync with conda-forge later. (the downside being, it's unsightly and distracting)

Reply:
No problem.  I'll just note that psubs have been removing `.ci_support` stuff from packages recently (only two but...).